src/components/Form.js

```diff
@@ -13,13 +13,18 @@ const Form = () => {
   const handleAuthor = (event) => {
     setAuthor(event.target.value);
   };
+  const [category, setCategory] = useState('');
+  const handleCategory = (event) => {
+    setCategory(event.target.value);
+  };
   const handleAddBook = () => {
     const newBook = {
       name: title,
       author,
+      category,
       id: Math.ceil(Math.random() * 10000),
     };
-    dispatch(addBook(newBook));
+    dispatch(addBooksAsync(newBook));
```

Comment:
- [ ] I tried to add a book. When I reloaded the page, I was not able to find the book that I added. Please make sure that the book is sent to the server using the API after clicking on the `Add book` button, and all the books should be fetched from the API while loading the page.